.eslintrc

```diff
@@ -1,3 +1,4 @@
 {
   "extends": "airbnb/base",
+  "parser": "babel-eslint"
```

Comment:
In this case I think is is an exception as the addition of this and the `plugins` line in the gruntfile is too allow the new code to work.  It would make sense from a historical standpoint to add dependencies that don't do anything in one PR and then start using them in a different PR?

I think this might be more of a debate into how to use git...